forge-gui/res/cardsfolder/upcoming/invasion_of_tarkir_defiant_thundermaw.txt

```diff
@@ -0,0 +1,26 @@
+Name:Invasion of Tarkir
+ManaCost:1 R
+Types:Battle Siege
+Defense:5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigReveal | TriggerDescription$ When CARDNAME enters the battlefield, reveal any number of Dragon cards from your hand. 
```

Comment:
I think `When you do, CARDNAME deals X plus 2 damage to any other target, where X is the number of cards revealed this way. (X can be 0.)` should be in this TriggerDescription too